src/views/Configurations/SmsConfig.vue

```diff
@@ -65,11 +71,17 @@
                                   class="custom"
                                   v-model="configData.cancelMessage"
                                   :value="configData.cancelMessage"
+                                  label="Appointment Message" 
+                                  label-placement="floating"
```

Comment:
This should read as in the place holder
```suggestion
                                  label="Cancel appointment Message" 
```